components/Form/Checkboxes/StyledCheckbox/StyledCheckbox.module.scss

```diff
@@ -29,14 +29,43 @@
     }
 
     &:hover::before {
-        background-color: rgba(26, 188, 156, 0.5);
-    }
-
-    &.checked::before {
-        background-color: #1abc9c;
+        opacity: 0.5;
```

Comment:
Instead of opacity we should use color here. Opacity's issue is it changes based on the background, so it will have different effects on light vs dark backgrounds. Also, the hover isn't very clear because of this. Probably some kind of orange so it contrasts the rest, ideally take this from the text color of the text inputs. Should color the box & x.

Also, make hover transitioned like the rest.